scripts/generate-credentials-with-sts-assume-role.sh

```diff
@@ -4,7 +4,6 @@ set -Eeou pipefail
 
 # This script uses aws sts assume-role to generate temporary credentials
 # and outputs them in $GITHUB_OUTPUT so those can be used in other workflow jobs.
-# role-arn = arn:aws:iam::358363220050:role/terraform-provider-mongodbatlas-acceptancetests
```

Comment:
role names and aws account id are not sensistive but we may want to consider to move this role name from a GH variable to secret

Reply:
no need to move it and yes it's not sensitive - just removing it from there for code hygiene